graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlErrorHandler.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.graphql;
+
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.common.annotation.UnstableApi;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+import graphql.ExecutionInput;
+import graphql.ExecutionResult;
+
+/**
+ * A handler that maps {@link ExecutionResult#getErrors()} or the specified {@link Throwable}
+ * to an {@link HttpResponse}.
+ */
+@UnstableApi
+@FunctionalInterface
+public interface GraphqlErrorHandler {
+
+    /**
+     * Return the {@link DefaultGraphqlErrorHandler}.
```

Comment:
```suggestion
     * Return the default {@link GraphqlErrorHandler}.
```
... because `DefaultGraphqlErrorHandler` is a package-private class so we cannot expose it to the Javadoc.